On Solaris, we deliver meson for a single Python version only, which should be
fine, because meson_python invokes it directly via /usr/bin/meson rather than
importing it.

That said, the following requirements prevent building any Python variant we
don't have Meson for. As such, these are removed and a manual dependency is
added to the manifest directly.

--- meson_python-0.18.0/pyproject.toml
+++ meson_python-0.18.0/pyproject.toml
@@ -6,8 +6,6 @@
 build-backend = 'mesonpy'
 backend-path = ['.']
 requires = [
-  'meson >= 0.64.0; python_version < "3.12"',
-  'meson >= 1.2.3; python_version >= "3.12"',
   'packaging >= 23.2',
   'pyproject-metadata >= 0.9.0',
   'tomli >= 1.0.0; python_version < "3.11"',
@@ -35,8 +33,6 @@ classifiers = [
 ]
 
 dependencies = [
-  'meson >= 0.64.0; python_version < "3.12"',
-  'meson >= 1.2.3; python_version >= "3.12"',
   'packaging >= 23.2',
   'pyproject-metadata >= 0.9.0',
   'tomli >= 1.0.0; python_version < "3.11"',
